mlflow/server/js/src/components/ParallelCoordinatesPlotPanel.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { ParallelCoordinatesPlotPanel } from './ParallelCoordinatesPlotPanel';
+
+describe('unit tests', () => {
+  let wrapper;
+  let mininumProps;
+
+  beforeEach(() => {
+    mininumProps = {
+      runUuids: ['runUuid_0', 'runUuid_1'],
+      sharedParamKeys: ['param_0', 'param_1'],
+      sharedMetricKeys: ['metric_0', 'metric_1'],
+    };
+  });
+
+  test('should render with minimal props without exploding', () => {
```

Comment:
Can we add a small test for this piece of logic?
```
        {(!_.isEmpty(selectedParamKeys) || !_.isEmpty(selectedMetricKeys)) ? (
```

Reply:
Sure, will add one.